ext/intl/common/common_date.cpp

```diff
@@ -139,7 +139,7 @@ U_CFUNC int intl_datetime_decompose(zval *z, double *millis, TimeZone **tz,
 		php_date_obj *datetime;
 		datetime = Z_PHPDATE_P(z);
 		if (!datetime->time) {
-			spprintf(&message, 0, "%s: the DateTime object is not properly "
+			spprintf(&message, 0, "%s: the DateTimeInterface object is not properly "
```

Comment:
Referring to a "DateTimeInterface object" is odd. I would suggest printing `ZSTR_VAL(Z_OBJCE_P(z)->name)` in here.